draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -145,6 +155,24 @@ const TooltipContent = ({
   ) : null
 }
 
+const renderChildren = (
+  content: ReactNode,
+  tooltipId: string,
+  hasActiveTooltip: boolean
+): ReactNode => {
+  if (isSemanticElement(content)) {
+    return cloneElement(content, {
+      "aria-describedby": hasActiveTooltip ? tooltipId : undefined,
+    })
+  }
+  /* eslint no-console: "warn" */
+  // We don't want to block them from this but just provide context for better a11y guidance
+  console.warn(
+    "<Tooltip /> is wrapping a non-semantic element, aria-describedby may not be read as expected"
```

Comment:
Suggestion for making the implications a bit more clear. Plus we can say for sure it won't work because we're now not adding `aria-describedby` if it doesn't pass this condition.

I've also added a mention of how the element should be focusable as well to ensure keyboard accessibility.

```suggestion
    "<Tooltip /> is not directly wrapping a semantic element, screen reader users will not be able to access the tooltip info. To ensure accessibility, Tooltip should be wrapping a semantic and focusable element directly."
```

Reply:
Good catch 👍 I'll also make sure to draw attention to it when this gets announces in #updates_design_system